### PR TITLE
remove storage of region heartbeat

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: Yang Keao <keao.yang@yahoo.com>

### CODE REVIEW

**Comment by YangKeao** [comment]
@Connor1996 Do you mean bootstrap?

File: scheduler/server/server.go [hunk 1]
```diff
@@ -394,15 +385,7 @@ func (s *Server) bootstrapCluster(req *pdpb.BootstrapRequest) (*pdpb.BootstrapRe
...
-	err = s.storage.SaveRegion(req.GetRegion())
-	if err != nil {
-		log.Warn("save the bootstrap region failed", zap.Error(err))
-	}
-	err = s.storage.Flush()
-	if err != nil {
-		log.Warn("flush the bootstrap region failed", zap.Error(err))
-	}
-	if err := s.cluster.start(); err != nil {
+	if err := s.cluster.start(WithRegion(core.NewRegionInfo(req.GetRegion(), nil))); err != nil {
```
Comment (Connor1996) [inline scheduler/server/server.go]: why need to start with region?

File: scheduler/client/client_test.go [hunk 2]
```diff
@@ -132,19 +132,9 @@ func newHeader(srv *server.Server) *pdpb.RequestHeader {
...
-		RegionEpoch: &metapb.RegionEpoch{
-			ConfVer: 1,
-			Version: 1,
-		},
-		Peers: peers[:1],
-	}
 	req := &pdpb.BootstrapRequest{
 		Header: header,
 		Store:  stores[0],
-		Region: region,
```
Comment (Connor1996) [inline scheduler/client/client_test.go]: remove the field from the proto

Comment (YangKeao) [reply scheduler/client/client_test.go]: OK!

**Comment by Connor1996** [comment]
Please fix the build

**Pull request merged**